### ISSUE TITLE
Bugs with TalkBack

### ISSUE THREAD

**Issue by charlag** [tutao/tutanota#2110]
v3.73.1
Report came in via Mastodon from the visually impaired user about the problems in a11y on Android.
1. When opening the menu, it is not focused. App continues to go through the emails and bottom navigation even though the menu is opened.
2. When opening the app the whole web title with "get your ad-free mailbox" is read.
3. With automatic login the progress dialog is not focused so there's no indication that login is happening.
4. Buttons in menu strip are not focusable in normal order (after folders), they are focusable after bottom navigation which should be hidden when menu is open anyway
5. Menu button is called "folders menu" even though we have other options there

**Comment by charlag** [comment]
Some are fixed with #8398
